contributing/community/speaker-mentoring.rst

```diff
@@ -0,0 +1,44 @@
+Speaker Mentoring
+=================
+
+The Symfony community benefits greatly when as many people as possible
+sharing their knowledge and experience with others. Every different
+point of view adds to our collective understanding of how to best use
+and evolve the code, design patterns and architecture provided within
+the Symfony community. Because of this, we specifically want to hear
+from long-time contributors nd new users, who often come across entirely
+different challenges with a totally fresh new look and perspective.
+
+How to get started
+------------------
+
+Giving a first talk at a conference can seem quite intimidating. But
+don't worry! At one time, every speaker went through the same process.
+And so, we want to make sure that as many people as possible are empowered
+to take this path if they are motivated. We have collected a few resources
+with advice to get started. More importantly, we can connect experienced
+speakers with people who are just taking their first steps in this area:
+
+.. tip::
+
+    A good first step might be to give a talk at a local user group to a
+    smaller crowd that one knows more intimately. A next step could be to
+    give a talk at conference in your mother tongue.
```

Comment:
`mother tongue` -> `first language` (it's the preferred term by Wikipedia too: https://en.wikipedia.org/wiki/First_language)